android/settings.gradle

```diff
@@ -37,3 +37,7 @@ includeUnimodulesProjects(
     ]
 // WHEN_DISTRIBUTING_REMOVE_TO_HERE
 )
+
+// Include Expo modules that are not unimodules
+include(":expo-random")
+project(":expo-random").projectDir = new File("../packages/expo-random/android")
```

Comment:
Would you please move it to nearer `include ':unimodules-test-core'` etc.? This should be wrapped with `// WHEN_DISTRIBUTING_REMOVE_{FROM,TO}_HERE`, otherwise it'll end up in Android shell app (Turtle shouldn't try to use this project, instead, `expokit` prebuilt AARs are available there).